frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { OverviewItem, Status } from '@console/shared';
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
+import { ListGroup } from 'patternfly-react';
+import { Grid, GridItem } from '@patternfly/react-core';
+import {
+  SidebarSectionHeading,
+  ResourceLink,
+  resourcePath,
+} from '@console/internal/components/utils';
+import { PipelineRunModel, PipelineModel } from '@console/dev-console/src/models';
+import { Link } from 'react-router-dom';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import { pipelineRunFilterReducer } from '../../../utils/pipeline-filter-reducer';
+import TriggerLastRunButton from './TriggerLastRunButton';
+
+const noOfPipelineRunsToshow = 3;
+
+type PipelineRunItemProps = {
+  pipelineRun: K8sResourceKind;
+};
+
+export const PipelineRunItem: React.FC<PipelineRunItemProps> = ({ pipelineRun }) => {
+  const {
+    metadata: { name, namespace, creationTimestamp },
+    status: { completionTimestamp, startTimestamp },
+  } = pipelineRun;
+  const path = resourcePath(referenceForModel(PipelineRunModel), name, namespace);
+  const lastUpdated = completionTimestamp || startTimestamp || creationTimestamp;
+  return (
+    <li className="list-group-item">
+      <Grid gutter="md">
+        <GridItem span={6}>
+          <Link to={`${path}`}>{name}</Link>
+          {lastUpdated && <span className="text-muted">&nbsp;({fromNow(lastUpdated)})</span>}
+        </GridItem>
+        <GridItem span={3}>
+          <Status status={pipelineRunFilterReducer(pipelineRun)} />
+        </GridItem>
+        <GridItem span={3} className="text-right">
+          <Link to={`${path}/logs`}>View Logs</Link>
+        </GridItem>
+      </Grid>
+    </li>
+  );
+};
+
+type PipelinesOverviewProps = {
+  item: OverviewItem & {
+    pipeline: K8sResourceKind[];
+    pipelineRuns: K8sResourceKind[];
+  };
+};
```

Comment:
Just an FYI, we are going to have overlap / conflict here. My PR for the tooltips has now been merged, so you'll probably want to rebase and adopt the changes I made to simplify things.

- Renamed `pipeline` to `pipelines` as it's an array - [REF](https://github.com/openshift/console/pull/3095/files#diff-bbb2e3628eb7d5b19c6b638c0968dc9dR31)
- Made this type in my PR - probably needs a rename to make use for both of us though - [REF](https://github.com/openshift/console/pull/3095/files#diff-5f5edb39a5aa09e8825496aa61c85f76R65)

Reply:
fixed in https://github.com/openshift/console/pull/3056/commits/8e27cbfc8c376c202bd2471c788bf0401b8dcb9d